GDJS/tests/tests/variable.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Common tests for gdjs game engine.
+ * See README.md for more information.
+ */
+
+describe('gdjs.variable', function() {
+
+	it('should parse initial values into strings and numbers', function(){
+
+		var int_var = new gdjs.Variable( {value : "526"} );
```

Comment:
Could you change the variable notation from snake_case to camelCase (`intVar`...)? Thanks! :)